Silicon/Ampere/AmpereAltraPkg/AmpereAltraPkg.dsc.inc

```diff
@@ -12,15 +12,8 @@
 [BuildOptions.common.EDKII.DXE_RUNTIME_DRIVER]
   GCC:*_*_AARCH64_DLINK_FLAGS = -z common-page-size=0x10000
 
-[BuildOptions]
-  GCC:RELEASE_*_*_CC_FLAGS  = -DMDEPKG_NDEBUG
-
 [LibraryClasses.common]
-!if $(TARGET) == RELEASE
-  DebugLib|MdePkg/Library/BaseDebugLibNull/BaseDebugLibNull.inf

```

Comment:
Set gEfiMdePkgTokenSpaceGuid.PcdDebugPrintErrorLevel to 0x80000002 in case of RELEASE build

Reply:
I've done that, but without this change you still don't get any output.

Reply:
yes, with enabling the serial print for the RELEASE, we should just enable for WARNING and ERROR by setting PcdDebugPrintErrorLevel to 0x80000002